debian/control

```diff
@@ -0,0 +1,33 @@
+Source: ttyd
+Section: utils
+Priority: optional
+Maintainer: Boyuan Yang <073plan@gmail.com>
+Build-Depends:
+ debhelper (>= 10),
+ cmake,
+ pkg-config,
+ libwebsockets-dev,
+ libjson-c-dev,
+ libssl-dev,
+ xxd,
```

Comment:
The `xxd` package only exists in [stretch (testing)](https://packages.debian.org/search?keywords=xxd) right now, this means the new package won't work for `wheezy` or `jessie` ?